dockers/Screenshotter/screenshotter.js

```diff
@@ -109,11 +67,13 @@ if (opts.exclude) {
     });
 }
 
-let seleniumURL = opts.seleniumURL;
-let seleniumIP = opts.seleniumIP;
-let seleniumPort = opts.seleniumPort;
-let katexURL = opts.katexURL;
-let katexIP = opts.katexIP;
+const browser = opts.browser || "firefox";
+const numAttempts = opts.attempts || 5;
+let seleniumURL = opts.seleniumUrl;
+let seleniumIP = opts.seleniumIp;
+let seleniumPort = opts.seleniumPort || 4444;
```

Comment:
You can pass the default value as the third argument or fourth argument(if collector function is present).

Reply:
Indeed, you're right.  Thanks!  That wasn't obvious to me from the documentation...

Reply:
There's actually a bug in the four-argument case: https://github.com/tj/commander.js/issues/686. But I was able to work around it.